src/v3/src/util/idxUtils.ts

```diff
@@ -145,8 +146,12 @@ export const areTransactionsEqual = (
     return false;
   }
 
-  const tx1AuthId = tx1?.context.currentAuthenticator?.value?.id;
-  const tx2AuthId = tx2?.context.currentAuthenticator?.value?.id;
+  const tx1AuthId = typeof tx1 !== 'undefined'
+    ? getCurrentAuthenticator(tx1)?.value?.id
+    : undefined;
+  const tx2AuthId = typeof tx2 !== 'undefined'
+    ? getCurrentAuthenticator(tx2)?.value?.id
+    : undefined;
   if (tx1AuthId !== tx2AuthId) {
```

Comment:
`undefined !== undefined` evaluates to `false` just making sure this is OK

Reply:
Yessir, this is expected because if they are both `undefined` it will ultimately fall to the `return true` block. Each condition checks for a difference between the values, if that condition results in `true` it will return false for the overall function.